src/OrbitQt/main.cpp

```diff
@@ -266,7 +266,8 @@ int main(int argc, char* argv[]) {
   absl::SetFlagsUsageConfig(absl::FlagsUsageConfig{{}, {}, {}, &orbit_core::GetBuildReport, {}});
   absl::ParseCommandLine(argc, argv);
 
-  InitLogFile(Path::GetLogFilePathAndCreateDir());
+  RemoveOldLogFiles(Path::CreateOrGetLogDir());
```

Comment:
Is this the only place where CreateOrGetLogDir is used? if so maybe remove it from Path.h and call in RemoveOldLogFiles implementation?

(if you need this parameter for a test - you can probably make it optional, so that by default the implemenation calls CreateOrGetLogDir())

Reply:
Thanks for the suggestion. `CreateOrGetLogDir` is also used in `GetLogFilePath`. I split the original `Path::GetLogFilePathAndCreateDir()` into two parts, i.e., `Path::CreateOrGetLogDir` and `Path::GetLogFilePath`. Therefore, I prefer to keep this method in `Path.h`.